Tests/fixtures/smartphone-18.yml

```diff
@@ -3273,3 +3273,327 @@
+  os:
+    name: Android
+    version: "11"
+    platform: ""
+  client:
+    type: browser
+    name: Chrome Mobile
+    version: 107.0.0.0
+    engine: Blink
+    engine_version: 107.0.0.0
+  device:
+    type: smartphone
+    brand: Starlight
+    model: Milan Pro
+  os_family: Android
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; Android 8.0.0; MobileMapper60) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Mobile Safari/537.36
+  os:
+    name: Android
+    version: 8.0.0
+    platform: ""
+  client:
+    type: browser
+    name: Chrome Mobile
+    version: 107.0.0.0
+    engine: Blink
+    engine_version: 107.0.0.0
+  device:
+    type: smartphone
+    brand: Trimble
+    model: MobileMapper 60
+  os_family: Android
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; Android 10; EC55) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.0.0 Mobile Safari/537.36
+  os:
+    name: Android
+    version: "10"
+    platform: ""
+  client:
+    type: browser
+    name: Chrome Mobile
+    version: 104.0.0.0
+    engine: Blink
+    engine_version: 104.0.0.0
+  device:
+    type: smartphone
+    brand: Zebra
+    model: EC55
+  os_family: Android
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; Android 7.1.2; EA602) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.62 Mobile Safari/537.36
+  os:
+    name: Android
+    version: 7.1.2
+    platform: ""
+  client:
+    type: browser
+    name: Chrome Mobile
+    version: 93.0.4577.62
+    engine: Blink
+    engine_version: 93.0.4577.62
+  device:
+    type: smartphone
+    brand: Unitech
+    model: EA602
+  os_family: Android
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; Android 8.1.0; E0111) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.73 Safari/537.36
+  os:
+    name: Android
+    version: 8.1.0
+    platform: ""
+  client:
+    type: browser
+    name: Chrome
+    version: 99.0.4844.73
+    engine: Blink
+    engine_version: 99.0.4844.73
+  device:
+    type: smartphone
+    brand: Everis
+    model: E0111
+  os_family: Android
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; Android 9; E0113) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36
+  os:
+    name: Android
+    version: "9"
+    platform: ""
+  client:
+    type: browser
+    name: Chrome
+    version: 107.0.0.0
+    engine: Blink
+    engine_version: 107.0.0.0
+  device:
+    type: smartphone
+    brand: Everis
+    model: E0113
+  os_family: Android
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; Android 9; E0114) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36
+  os:
+    name: Android
+    version: "9"
+    platform: ""
+  client:
+    type: browser
+    name: Chrome
+    version: 110.0.0.0
+    engine: Blink
+    engine_version: 110.0.0.0
+  device:
+    type: smartphone
```

Comment:
tablet https://www.productreview.com.au/listings/everis-android-90-tablet-e0114

Reply:
All Everis devices were tablets. I've fixed the other too.